client/components/editor/structure/TesList.vue

```diff
@@ -44,60 +42,52 @@
 import AddElement from 'tce-core/AddElement';
 import Draggable from 'vuedraggable';
 import InlineActivator from './InlineActivator';
-import last from 'lodash/last';
+
+const draggableConfig = {
+  handle: '.drag-handle',
+  scrollSpeed: 15,
+  scrollSensitivity: 125
+};
 
 export default {
   props: {
-    list: { type: Array, default() { return []; } },
-    dragOptions: { type: Object, default() { return {}; } },
+    list: { type: Array, default: () => [] },
+    dragOptions: { type: Object, default: () => ({}) },
     enableAdd: { type: Boolean, default: true },
     types: { type: Array, default: null },
     activity: { type: Object, required: true },
     layout: { type: Boolean, default: false },
     embedded: { type: Boolean, default: false }
   },
-  data() {
-    return {
-      dragElementIndex: -1,
-      insertPosition: 0,
-      isElementDrawerVisible: false
-    };
-  },
+  data: () => ({
+    dragElementIndex: -1,
+    insertPosition: 0,
+    showElementDrawer: false
+  }),
   computed: {
-    options() {
-      return Object.assign(this.dragOptions, {
-        handle: '.drag-handle',
-        scrollSpeed: 15,
-        scrollSensitivity: 125
-      });
-    },
-    lastPosition() {
-      const lastItem = last(this.list);
-      return lastItem ? lastItem.position + 1 : 1;
-    }
+    options: ({ dragOptions }) => ({ ...dragOptions, ...draggableConfig }),
```

Comment:
Can we check if the `dragOptions` is ever used? As @abasic mention in #350, if prop isn't use, there is no point of declaring it. Cause we use the the `draggableConfig` anyway.